src/main/java/org/testng/remote/strprotocol/GenericMessage.java

```diff
@@ -37,25 +37,6 @@ public void setTestCount(int testCount) {
     m_testCount = testCount;
   }
 
-//  public GenericMessage(final int type, Map props) {
-//    m_messageType = type;
-//    m_properties = props;
-//  }
-
-//  public GenericMessage addProperty(final String propName, final Object propValue) {
-//    m_properties.put(propName, propValue);
-//
-//    return this;
-//  }
-
-  public GenericMessage addProperty(final String propName, final int propValue) {
```

Comment:
Why do you remove this method? It could be used by testng extension like ide plugins.